bundles/org.openhab.binding.homematic/src/test/java/org/openhab/binding/homematic/internal/communicator/virtual/ButtonDatapointTest.java

```diff
@@ -16,9 +16,11 @@
 import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
This code should better be reviewed by someone who knows it better. I'm not 100% sure if the double click can really be tested cleanly automatically, because the timing is critical.

Reply:
Removed the double press handling (and respective unit test) for the reasons outlined [below](https://github.com/openhab/openhab-addons/pull/11186#issuecomment-921548347)